toolchain/install/BUILD

```diff
@@ -205,6 +205,14 @@ manifest(
     srcs = [":install_data"],
 )
 
+# A list of clang's installed builtin header files.
+# This is consumed by //toolchain/testing:file_test.
+manifest(
+    name = "clang_headers_manifest.txt",
+    srcs = [":lib/carbon/llvm/lib/clang/" + LLVM_VERSION_MAJOR + "/include.pkg"],
```

Comment:
Am I missing something about how this works to get the list of all the files?

It seems to work with `stddef.h` in the tests, so I'm guessing I'm missing something here...

Ah, this will be an internal rule from `make_install_filegroups` I would guess. That seems... A bit subtle. But I also don't really have a better idea. Are these rules already a documented output? (I don't have the full checkout ready to hand to read, sorry for maybe obvious question.)